### PR TITLE
pscale: add shell completion and version test

### PR DESCRIPTION
**Pull request by superherointj**
pscale: add shell completion and version test

### CODE REVIEW

**Comment by superherointj** [comment]
Result of `nixpkgs-review pr 189811` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>pscale</li>
  </ul>
</details>

File: pkgs/development/tools/pscale/default.nix [hunk 1]
```diff
@@ -20,10 +26,23 @@ buildGoModule rec {
     "-X main.date=unknown"
   ];
 
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+    installShellCompletion --cmd ${pname} \
+      --bash <($out/bin/{pname} completion bash) \
+      --fish <($out/bin/{pname} completion fish) \
+      --zsh <($out/bin/{pname} completion zsh)
```
Comment (SuperSandro2000) [inline pkgs/development/tools/pscale/default.nix:35]:
```suggestion
    installShellCompletion --cmd pscale \
      --bash <($out/bin/pscale completion bash) \
      --fish <($out/bin/pscale completion fish) \
      --zsh <($out/bin/pscale completion zsh)
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/development/tools/pscale/default.nix (hunk 1 above)
Comment (superherointj) [reply pkgs/development/tools/pscale/default.nix]: Done

**Comment by superherointj** [comment]
Thanks!

**Pull request merged**